examples/teleop/package-lock.json

```diff
@@ -10,7 +10,7 @@
       "license": "Apache-2.0",
       "dependencies": {
         "@improbable-eng/grpc-web": "^0.15.0",
-        "@viamrobotics/sdk": "^0.0.21"
+        "@viamrobotics/sdk": "file:../../../typescript-sdk/viamrobotics-sdk-0.0.27.tgz"
```

Comment:
Unfortunately, I will need to push a follow-up commit after this PR is merged and the new version is live.